package.json

```diff
@@ -15,8 +15,8 @@
     "clean-all": "node ./scripts/clean-all.js",
     "vrtest": "cd apps && cd vr-tests && npm run screener",
     "a11ytest": "cd apps && cd a11y-tests && npm run test",
-    "start": "cd apps && cd fabric-website-resources && npm start",
-    "start-exp": "cd packages && cd experiments && npm start",
+    "start": "yarn workspace @uifabric/fabric-website-resources start",
```

Comment:
What does this do? Should we be doing it for other commands too?

Reply:
we should - we couldn't pass any args if we used the && syntax.